### ISSUE TITLE
Responsives Theme

### ISSUE THREAD

**Issue by wintstar** [coppermine-gallery/cpg1.6.x#26]
I really appreciate her work. A very good job. Many are now online with the mobile devices. I am aware that there is a mobile theme for cpg. Most of them are no longer up to date and also not very good. Will there be a responsive standard theme with the next release?

**Comment by ron4mac** [comment]
No, there will not be a standard responsive theme soon with any release. The development team is too small and a great deal of work would be necessary before a truly responsive theme could be possible. for now, the best alternative is the [Mobile Light](url
http://forum.coppermine-gallery.net/index.php/topic,79540.0.html) theme in conjunction with [Theme Switcher](url)http://forum.coppermine-gallery.net/index.php/topic,79377.0.html).

**Comment by wintstar** [comment]
That's understandable. I wish you a Merry Christmas.
